tutoriale/Instalacja_Anakondy_W10-11.md

```diff
@@ -0,0 +1,56 @@
+# Anaconda
+
+Anaconda to popularna platforma Data Science, która rozszerza funkcjonalność podstawowego języka Python. Łączy ona narzędzia potrzebne do analizy danych, projektów big data, tworzenia modeli predykcyjnych i wizualizacji danych np. ich prezentowanie na wykresach. Umożliwia wykorzystywanie bibliotek z różnych języków np. Python, R.
+
+Wraz z Anacondą zainstalujesz:
+* edytor Jupyter Notebook
+* edytor tekstu Spyder
+* biblioteki Numpy, Pandas, Matplotlib, scikit-learn, SciPy i szereg innych
+* menadżer pakietów Conda – jest to własny menadżer pakietów Anacondy i działa podobnie jak PIP w Pythonie podstawowym
+
+Listę wszystkich dostępnych bibliotek, które zostaną zainstalowane wraz z Anacondą znajdziesz pod następującym linkiem 
+<a href="https://docs.anaconda.com/free/anaconda/reference/packages/pkg-docs/">Anaconda Pakiety</a>.
+
+## Instalacja Condy
+
+Aby zainstalować Anacondę na swoim komputerze musimy najpierw pobrać instalator, który znajdziemy pod poniższym linkiem 
+<a href="https://www.anaconda.com/download">Anaconda - pobieranie</a>, który przekieruje nas bezpośrednio do strony pobierania. Dalej pobieramy instalator.
+
+![](attachment:Anaconda%20download.png)
```

Comment:
Ten link oczywiście nie działa

Reply:
z tego co sprawdziłem jeśli wyeksportuje się notatnik z Jupytera w formacie .ipynb i wrzuci na githuba to obrazki się wyświetlają bez dodawania jpgów do repo.  Czy w związku z tym mogę zamienić ten plik .md na plik .ipynb?